routing/missioncontrol.go

```diff
@@ -50,6 +50,11 @@ const (
 	// DefaultAprioriWeight is the default a priori weight. See
 	// MissionControlConfig for further explanation.
 	DefaultAprioriWeight = 0.5
+
+	// DefaultMinFailureRelaxInterval is the default minimum time that must
+	// have passed since the previously recorded failure before the failure
+	// amount may be raised.
+	DefaultMinFailureRelaxInterval = time.Minute
```

Comment:
If we are not making this configurable, we should remove `Default` from the name.

Reply:
The idea is that this isn't a default for the user, but a default for other subsystems using mission control.